lib/ansible/plugins/strategy/linear.py

```diff
@@ -252,7 +252,7 @@ def run(self, iterator, play_context):
 
                         run_once = templar.template(task.run_once) or action and getattr(action, 'BYPASS_HOST_LOOP', False)
 
-                        if (task.any_errors_fatal or run_once) and not task.ignore_errors:
+                        if (play_context.any_errors_fatal or run_once) and not task.ignore_errors:
```

Comment:
this prevents the play setting from being overriden by block/role/tasks/etc.

Reply:
This seems to work now for blocks, roles, and tasks. @bcoca When you have time, can you look at this again?

Reply:
from the changes this will now be ignored when set at role/block or task level, only play level is valid (rest are still settable, but ignored)